docs/references/strategies/languages/python/setuptools.md

```diff
@@ -11,6 +11,19 @@ approach to dependency management in python projects.
 
 ## Analysis: requirements.txt
 
+For the CLI to identify transitive dependencies from a requirements.txt file,
```

Comment:
[Prefer] We include some text around commands we are using to infer deep dependencies: `pip show`, etc.